.github/pull_request_template.md

```diff
@@ -1,18 +1,15 @@
 Following this checklist to help us incorporate your
 contribution quickly and easily:
 
- - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MNG) filed
-       for the change (usually before you start working on it).  Trivial changes like typos do not
-       require a JIRA issue. Your pull request should address just this issue, without
-       pulling in other changes.
- - [ ] Each commit in the pull request should have a meaningful subject line and body.
- - [ ] Format the pull request title like `[MNG-XXX] SUMMARY`, where you replace `MNG-XXX`
-       and `SUMMARY` with the appropriate JIRA issue. Best practice is to use the JIRA issue
-       title in the pull request title and in the first line of the commit message.
- - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will
-       be performed on your pull request automatically.
- - [ ] You have run the [Core IT][core-its] successfully.
+- [ ] Your pull request should address just one issue, without pulling in other changes.
+- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
+- [ ] Each commit in the pull request should have a meaningful subject line and body.
+  Note that commits might be squashed by a maintainer on merge.
+- [ ] Write unit tests that match behavioral changes, where the tests fail if the changes to the runtime are not applied.
+  This may not always be possible but is a best-practice.
+- [ ] Run `mvn verify` to make sure basic checks pass.
+  A more thorough check will be performed on your pull request automatically.
+- [ ] You have run the integration tests successfully (`mvn -Prun-its verify`).
```

Comment:
```suggestion
- [ ] You have run the integration tests successfully (`mvn verify -Prun-its`).
```
goal (more) important, profile is context.

Reply:
BTW: they dont run on my system.
just like main build fails mostly with 3 test reaching 15 sec. timeout.

Reply:
As stated in the main PR the profile does not exists, so it fails